keyboards/anavi: Add ANAVI Macro Pad 8 (#9164)
qmk/qmk_firmware: keyboards/anavi/macropad8/readme.md

# nano

> ![nano](https://1.bp.blogspot.com/-goa_eXx1McA/WEhvoSdfDSI/AAAAAAAB_bM/XdstcXe_eKkQxuYcNI5cTySTVo3xZxaeQCEw/s640/IMG_20161207_101051.jpg)

Reviewer: The nano actaully already exists in the repo (under `/keyboards/40percentclub/nano`). 

If this is just a modification of that, then this would be better as a keymap under that, rather than as a separate board.

# ANAVI Macro Pad 8

ANAVI Macro Pad 8 is an open source mini mechanical keyboard with 8 keys, backlit, addressable RGB WS2812B LED strip on the back and mini OLED display. Powered by ATmega32U4 microcontroller and with microUSB connector.

> * Keyboard Maintainer: Leon Anavi

Reviewer 2: ```suggestion
* Keyboard Maintainer: [Leon Anavi](https://github.com/leon-anavi)
```

* Hardware Supported: ANAVI Macro Pad 8

> * Hardware Availability: [PCB Files](https://github.com/AnaviTechnology/anavi-macro-pad-8)

Reviewer 2: Just a heads-up, this link is currently Error 404.

Author: Ops, sorry, I was a private repo, I made it public. Thanks, Leon

# ANAVI Macro Pad 8

> ===
> 
> ANAVI Macro Pad 8 is an open source mini mechanical keyboard with 8 keys, backlit, addressable RGB WS2812B LED strip on the back and mini OLED display. Powered by ATmega 32U4 microcontroller and with microUSB connector.

Reviewer 3: ```suggestion

ANAVI Macro Pad 8 is an open source mini mechanical keyboard with 8 keys, backlit, addressable RGB WS2812B LED strip on the back and mini OLED display. Powered by ATmega32U4 microcontroller and with microUSB connector.
```

> Keyboard Maintainer: Leon Anavi
> Hardware Supported: ANAVI Macro Pad 8
> Hardware Availability: [PCB Files](https://github.com/AnaviTechnology/anavi-macro-pad-8)

Reviewer 3: ```suggestion
* Keyboard Maintainer: Leon Anavi
* Hardware Supported: ANAVI Macro Pad 8
* Hardware Availability: [PCB Files](https://github.com/AnaviTechnology/anavi-macro-pad-8)
```

> Make example for this keyboard (after setting up your build environment):  

Reviewer 3: ```suggestion
Make example for this keyboard (after setting up your build environment):
```
